Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py

```diff
@@ -704,8 +704,11 @@ def _run_single_test(self,
         :param test_spec: The test class that should be a subclass of SingleTest
         :return: None
         """
-        # Set the self.executable program for Launcher ad re-bind our param workspace to it.
-        self.executable = self._executable_function(workspace)
+        # Set the self.executable program for Launcher and re-bind our param workspace to it.
+        if self.atom_tools_executable_name:  # Atom Tools test.
+            self.executable = launcher_helper.create_atom_tools_launcher(workspace, self.atom_tools_executable_name)
+        else:  # Editor test.
+            self.executable = launcher_helper.create_editor(workspace)
```

Comment:
_executable_function was replaced with this logic in multiple places. Can it be passed in or configured once?

Reply:
all that needs to be set is the `self.atom_executable_name` on the `AtomToolsTestSuite` class, so yes